lib/modules/showImages.js

```diff
@@ -1666,6 +1689,51 @@ addModule('showImages', (module, moduleID) => {
 		}
 	}
 
+	async function setMediaControls(mediaTag) {
+		// Is bar enabled
+		if (!module.options.imageControls.value) return;
+
+		// Generate media controls
+		const template = await RESTemplates.load('mediaControls');
+		const controls = document.createElement('div');
+		controls.className = `RESMediaControls ${module.options.imageControlsPosition.value}`;
+		controls.innerHTML = template.text();
+
+		const $media = $(mediaTag);
+		$media.data('media-controls', controls);
+		$media.parent().append(controls);
+
+		// Hook up controls
+		$(controls).find('span').click(function(e) {
+			e.preventDefault();
+			e.stopPropagation();
+
+			// Implement control logic
+			switch ($(this).data('action')) {
+				case 'rotateLeft':
+					module.rotateMedia(mediaTag, ($media.data('rotation-state') || 0) - 1);
+					break;
+				case 'rotateRight':
+					module.rotateMedia(mediaTag, ($media.data('rotation-state') || 0) + 1);
+					break;
+				case 'imageLookup':			
+					let lookupUrl = ($media[0].tagName==='IMG') ? $media.attr("src") : $media.parent().attr('href');
+
+					// Google doesn't like image url's without a protacol
+					if (lookupUrl.indexOf('//') === 0){
+						lookupUrl = document.location.protocol + lookupUrl;
+					}
+
+					RESEnvironment.openNewTab(`https://images.google.com/searchbyimage?image_url=${lookupUrl}`, true);
```

Comment:
Minor nit: this will focus the tab by default, so the `true` argument can be omitted